release/air_tests/air_benchmarks/mlperf-train/app_config_oom.yaml

```diff
@@ -0,0 +1,14 @@
+base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_CPU"] | default("anyscale/ray-ml:2.0.1-py37-cpu") }}
+env_vars: {"RAY_task_oom_retries": "50", "RAY_DISABLE_MEMORY_MONITOR": "1", "RAY_memory_monitor_interval_ms": "100", "RAY_min_memory_free_bytes": "1000000000"}
```

Comment:
Just for my understanding, what is the memory monitor and why is it disabled?

Reply:
This is an older impl that blocks task submission if memory usage is high, we disable it since it could cause the test to fail since the node is constantly running close to memory limit

Reply:
Does `RAY_DISABLE_MEMORY_MONITOR` disable our oom killer?

So the test will oom and fail without our killer and succeed with it?

Reply:
Leaving the python mem monitor on means the test will be flakey.

if a task is submitted when node is low on memory the test will fail. So we disable that to prevent random failures

Reply:
Maybe add a comment so future readers aren't left with the same question

Reply:
What's the effect with and without oom killer for this test?

Reply:
test won't pass

test results: https://docs.google.com/document/d/1Hz7nhZ-BMwXYt2Y-1RkwhbOHq_39okSzG7YQwNN4evk/edit#heading=h.n0sptse2lmzn

Reply:
kk, so with oom killer, it will pass.